pip/pip-375.md

```diff
@@ -0,0 +1,58 @@
+# PIP-375: Expose the Admin client configs: readTimeout, requestTimeout, connectionTimeout
+
+# Background knowledge
+
+- `pulsar-admin sinks create` will upload files to brokers, but if the file is large and the network is slow, it may cost a long time.
+- The default value of `requestTimeout` of Admin API is `60s`, after the fix https://github.com/apache/pulsar/pull/23128, it was changed to `300s`.
+
+# Motivation
+
+We'd better to expose the following configs to users:
+
+```properties
+# The value is the timeout to build a connection to the server. Value 0 represents infinity. Negative values are not allowed.
+connectionTimeout
+
+# The value is the timeout to read a response. If the server doesn't respond within the defined timeframe, ProcessingException is thrown with TimeoutException as a cause.
```

Comment:
Is it possible that the server responds, but the client just doesn't know about this? 🤔
What if the server responds, but the response isn't fully received by the client? 
Is "response timeout" different from "read timeout"?
Could we improve the definition to reduce the confusion?

Reply:
It's better to have clear descriptions for the settings. You could use GitHub Copilot to revisit the comment based on the feedback to assist in coming up with an improvement.